src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+            aggregate: *Collection
+            pipeline: [{$project: {tanh: { $tanh: $double}}}]
+            cursor: {}
+
+- Name: ExpressionDegreesToRadians
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionDegreesToRadians
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {degreesToRadians: { $degreesToRadians: $double}}}]
+            cursor: {}
+
+- Name: ExpressionRadiansToDegrees
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionRadiansToDegrees
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {radiansToDegrees: { $radiansToDegrees: $double}}}]
+            cursor: {}
+
+- Name: ExpressionDayOfMonth
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionDayOfMonth
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {dayOfMonth: { $dayOfMonth: $date}}}]
+            cursor: {}
+
+- Name: ExpressionDayOfWeek
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionDayOfWeek
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {dayOfWeek: { $dayOfWeek: $date}}}]
+            cursor: {}
+
+- Name: ExpressionDayOfYear
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionDayOfYear
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {dayOfYear: { $dayOfYear: $date}}}]
```

Comment:
note that date doesn't exist in the schema; you only have date1 and date2 (here above and below)

Reply:
Oh yes! Done.